Kitodo-ImageManagement/src/main/java/org/kitodo/imagemanagement/ConvertRunner.java

```diff
@@ -51,7 +51,7 @@ class ConvertRunner {
     /**
      * Default timeout.
      */
-    private static final int DEFAULT_TIMEOUT_MINS = (int) TimeUnit.MINUTES.convert(2, TimeUnit.HOURS);
+    private static final int DEFAULT_TIMEOUT_SECS = (int) TimeUnit.SECONDS.convert(2, TimeUnit.HOURS);
```

Comment:
I wonder whether `7200`, `2 * 3600` or `2 * 60 * 60` would not be easier to read. At least it would avoid the type cast.

Reply:
The type cast could also be avoided by using a `long DEFAULT_TIMEOUT_SECS` and `KitodoConfig.getLongParameter` below.